src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double area;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.area = calculateArea();
```

Comment:
You're calculating the area in the constructor and storing it in a class field. This is unnecessary, as you can calculate the area on-demand in the 'getArea()' method.